thoth/storages/graph/models.py

```diff
@@ -50,6 +50,12 @@
     EnvironmentTypeEnum.RUNTIME.value, EnvironmentTypeEnum.BUILDTIME.value, name="environment_type", create_type=True
 )
 
+association_table = Table(
```

Comment:
this should be a class as all tables

Reply:
logic of connection between `ImportPackage` and `PPV` was changed to Association Object as requested by using `FoundImportPackage` in commit b4147fd5152048337263037bd70f41c6a96026af